src/cls/tabular/skyroot.fbs

```diff
@@ -0,0 +1,107 @@
+namespace Tables;
+
+union Relation { 
+  Rows,
+  Col,
+  ColInt,
+  ColFloat,
+  ColString
+}
+
+union Data { 
+  IntData,
```

Comment:
i think we can use SDT_INT etc types here, just for these 3 cases and add more later.

Reply:
Yup. I changed these identifiers in the new version of the fbs.